tools/clang/lib/SPIRV/SPIRVEmitter.cpp

```diff
@@ -2638,7 +2652,8 @@ SPIRVEmitter::processBufferTextureLoad(const CXXMemberCallExpr *expr) {
     return processBufferTextureLoad(object, coordinate, constOffset, varOffset,
                                     lod);
   }
-  emitError("Load() is not implemented for the given object type.");
+  emitError("Load() is not implemented for the given object type.",
```

Comment:
Load() of the given object type unimplemented

Reply:
Done